test/functional/ui/mouse_spec.lua

```diff
@@ -1913,14 +1913,28 @@ describe('ui/mouse/input', function()
     api.nvim_set_var('mouse_move2', 0)
     command('nnoremap <MouseMove> <Cmd>let g:mouse_move += 1<CR>')
     command('nnoremap <2-MouseMove> <Cmd>let g:mouse_move2 += 1<CR>')
-    feed('<MouseMove><0,0>')
-    feed('<MouseMove><0,0>')
-    api.nvim_input_mouse('move', '', '', 0, 0, 0)
-    api.nvim_input_mouse('move', '', '', 0, 0, 0)
+    feed('<MouseMove><1,0>')
+    feed('<MouseMove><2,0>')
+    api.nvim_input_mouse('move', '', '', 0, 0, 3)
+    api.nvim_input_mouse('move', '', '', 0, 0, 4)
```

Comment:
The description of this test should be changed. Maybe to this:
```
<MouseMove> to different locations can be mapped
```

Reply:
DONE